base/sparse/linalg.jl

```diff
@@ -941,3 +941,37 @@ end
 chol(A::SparseMatrixCSC) = error("Use cholfact() instead of chol() for sparse matrices.")
 lu(A::SparseMatrixCSC) = error("Use lufact() instead of lu() for sparse matrices.")
 eig(A::SparseMatrixCSC) = error("Use eigs() instead of eig() for sparse matrices.")
+
+function Base.cov(X::SparseMatrixCSC, vardim::Int=1; corrected::Bool=true)
+    a, b = size(X)
+    n, p = vardim == 1 ? (a, b) : (b, a)
+
+    # Cov(X) = E[(X-μ)'(X-μ)]
```

Comment:
Maybe extend to `# Cov(X) = E[(X-μ)'(X-μ)] = E(XX') - μμ'` and simplify the comments below